doc/tutorials/using_celadon_as_uos.rst

```diff
@@ -0,0 +1,109 @@
+
+Build Celadon from source
+*************************
+
+#. Follow the `Build Celadon from source
+   <https://01.org/projectceladon/documentation/getting_started/build-source>`_
+   to get the Celadon project source code ready
+
+#. Select Celadon build target::
+
+      $ cd <Celadon project directory>
+      $ source build/envsetup.sh
+      $ lunch cel_apl-userdebug
+
+   .. note:: You can run ``lunch`` with no arguments in order to choose your Celadon build variants.
+
+#. Apply these additional patches:
+
+   .. table:: ACRN patch list
+      :widths: auto
+      :name: ACRN patch list
+
+      +--------------------------------------------------------------------+-------------------------------------------+
+      | Patch link                                                         | Description                               |
+      +====================================================================+===========================================+
+      | https://github.com/projectceladon/device-androidia/pull/458        | kernel config: Add the support of ACRN    |
+      +--------------------------------------------------------------------+-------------------------------------------+
+      | https://github.com/projectceladon/device-androidia-mixins/pull/293 | graphic/mesa: Add the support of ACRN     |
+      +--------------------------------------------------------------------+-------------------------------------------+
+      | https://github.com/projectceladon/device-androidia/pull/441        | cel_apl: use ttyS0 instead of ttyUSB0     |
+      +--------------------------------------------------------------------+-------------------------------------------+
+      | https://github.com/projectceladon/device-androidia/pull/439        | Disable trusty and pstore                 |
+      +--------------------------------------------------------------------+-------------------------------------------+
+
+   .. note:: You may need to modify the source code manually if meet the error when use
+      ``git apply`` command to apply the patches above.
+
+#. Build Celadon image::
+
+   $ device/intel/mixins/mixin-update
+   $ make SPARSE_IMG=true gptimage -j $(nproc)
+
+.. note:: Replace the ``$(nproc)`` argument with the number of processor threads on your workstation
+   in order to build the source code with parallel tasks. The Celadon gptimage will be
+   generated to ``out/target/product/cel_apl/cel_apl_gptimage.img``
+
+Steps for Using Celadon as User OS
+**********************************
+
+#. Follow :ref:`getting-started-apl-nuc` to boot the "ACRN Service OS"
+
+#. Download ``OVMF.fd`` from `link
+   <https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/doc/tutorials/OVMF.fd>`__
+
+#. Download ``launch_android.sh`` script from `link
+   <https://raw.githubusercontent.com/tanminger/acrn-hypervisor/ed7a758377151b4c7296c0b5669167585b50907b/devicemodel/samples/nuc/launch_android.sh>`__
+
+#. Boot Celadon as User OS::
```

Comment:
Change to:
```
#. Use ``scp`` to copy these downloaded files from your development system to your NUC::
```
Is there a reason you can't download the files in the previous step  on the NUC directly instead of copying them from your development system?

Reply:
I was planing to let user download directly to the NUC from the repo in the steps, but the link is too long and looks strange, so I give up :smile:

Reply:
Well, it's the same link you have to use on the development system, or are you thinking you'd click the link in the documentation on your development system, but you'd have to type it in on your NUC?  I'm not sure what the issues is?

Reply:
Now it's verified that Celadon UOS is able to boot successfully with the build-in ``OVMF.fd`` and ``launch_win.sh`` script, I will update the steps in next release.